### PR TITLE
Bug 1777137: don't post new observed config on resources validation

### PR DESCRIPTION
**Pull request by stlaz**
let's use this PR to also pull the below lib-go fix so that the BZ is verifiable:
- https://github.com/openshift/library-go/pull/861

/hold
contains fake bump

### CODE REVIEW

**Comment by stlaz** [comment]
/hold cancel
removed fake bump, added prefixes to the observers

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -572,7 +572,7 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		operatorCtx.operatorClient.Informers,
 		eventRecorder)
 
-	auditPolicyPahGetter, err := libgoassets.NewAuditPolicyPathGetter()
+	auditPolicyPahGetter, err := libgoassets.NewAuditPolicyPathGetter("/var/run/configmaps/audit")
```
Comment (vareti) [inline pkg/operator/starter.go:575]: nit: not coming from your changes but maybe a good time to fix the variable name to `auditPolicyPathGetter`?

**Comment by vareti** [comment]
small nit. other looks good to me

**Comment by stlaz** [comment]
fixed the typo

**Comment by stlaz** [comment]
/retest
Looks like CI is slowly starting to catch on fire

**Pull request merged**